literature.bib

```diff
@@ -102,13 +168,13 @@ @inproceedings{MOP+:SPLC19
 	isbn = {9781450371384},
 	doi = {10.1145/3336294.3336297},
 	pages = {289--301},
-	keywords = {software product lines, model counting, numerical features, feature model, propositional formula, bit-blasting},
+	keywords = {software product lines, model counting, numerical features, extended feature model, feature model, non-functional properties, propositional formula, bit-blasting},
```

Comment:
these should be the keywords as written in the paper. have they been wrong?

Reply:
No, I just started at the wrong point. Removed my added tags.